packages/esm-service-queues-app/src/patient-search/visit-form/base-visit-type.component.tsx

```diff
@@ -12,9 +12,10 @@ interface BaseVisitTypeProps {
   onChange: (event) => void;
   patientUuid: string;
   visitTypes: Array<VisitType>;
+  enableSearch?: boolean;
 }
 
-const BaseVisitType: React.FC<BaseVisitTypeProps> = ({ onChange, visitTypes }) => {
+const BaseVisitType: React.FC<BaseVisitTypeProps> = ({ onChange, visitTypes, enableSearch = false }) => {
```

Comment:
I think the `enableSearch` is still a bit of a question mark.  Personally, I see no reason to have a search here.  But let's just leave the search as-is for now and we can assess more in the future.

Reply:
@mseaton Kindly could you please provide more insight into the reasons behind this opinion? Let's maintain the search feature as it is for now, and we can reassess its relevance and effectiveness in the future? I am a bit confused on the changes to be made basing on the review.

Reply:
@jwnasambu - issue with the search getting hidden is a real bug.  The issue with the search not being necessary is an opinion I am expressing, but not really a problem.  Given that the designs have the search, we can leave it for now and maybe start a conversation on the ux-design-advisory asking about the thinking behind the visit type search.  It would be good to make sure we fully understand what use case this is envisioned for (eg. are there implementations out there that have so many visit types we would need an auto-complete search for them;  who/what implementation is this designed for where this is a need?

Reply:
Beyond this, if we _are_ going to have this boolean `enableSearch` prop on the component, then we will need to also support this configurability in the config schema and ensure the configured value is getting passed in and used when this component is instantiated.  I don't see that in this PR, and it's not really priority for us to have this removed, so this is why I said just to leave it.